Src/PRuntimes/PExplicitRuntime/src/main/java/pexplicit/PExplicit.java

```diff
@@ -11,15 +26,149 @@ public class PExplicit {
 
     /**
      * Main entry point for PExplicit runtime.
-     * TODO
      */
     public static void main(String[] args) {
         // configure Log4J
         Log4JConfig.configureLog4J();
 
-        // get reflection to fetch pexplicit model IR
+        // parse the commandline arguments to create the configuration
+        PExplicitGlobal.setConfig(PExplicitOptions.ParseCommandlineArgs(args));
+        PExplicitLogger.Initialize(PExplicitGlobal.getConfig().getVerbosity());
+
+        // get reflections corresponding to the model
         Reflections reflections = new Reflections("pexplicit.model");
```

Comment:
What is this for?

Reply:
Used at 2 places in PExplicit.java:
1. To instantiate the IR (by looking into class that extends `PModel` abstract class in the jar)
2. To search for and instantiate the appropriate test driver (based on the input test driver provided with `-tc`)